hack/build/docker/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:28
+FROM fedora:29
 
 RUN dnf install -y qemu xz gzip git gradle gcc && dnf clean all
```

Comment:
Do you have to do the business with the repo file here?

Reply:
I did try that but it got confused when I did, I can try again see if I can do it differently.

Reply:
So I can't because the environment in which this docker file is run in different from the other so the soft link to the repo file in automation doesn't work.